config/tests/samples/create/harness.go

```diff
@@ -278,34 +278,37 @@ func MaybeSkip(t *testing.T, name string, resources []*unstructured.Unstructured
 	if os.Getenv("E2E_GCP_TARGET") == "mock" {
 		for _, resource := range resources {
 			gvk := resource.GroupVersionKind()
-			switch gvk.GroupKind() {
-			case schema.GroupKind{Group: "certificatemanager.cnrm.cloud.google.com", Kind: "CertificateManagerCertificate"}:
-			case schema.GroupKind{Group: "certificatemanager.cnrm.cloud.google.com", Kind: "CertificateManagerCertificateMap"}:
-			case schema.GroupKind{Group: "certificatemanager.cnrm.cloud.google.com", Kind: "CertificateManagerDNSAuthorization"}:
-				// ok
 
+			supported := false
+
+			switch gvk.Group {
+			case "certificatemanager.cnrm.cloud.google.com":
+				supported = true
+			}
+
+			switch gvk.GroupKind() {
 			case schema.GroupKind{Group: "iam.cnrm.cloud.google.com", Kind: "IAMServiceAccount"}:
-				// ok
+				supported = true
```

Comment:
Seems easier to use [continue](https://go.dev/play/p/KlMkFH_wtpF) rather than futzing with the supported flag.